.github/workflows/docs.yml

```diff
@@ -33,7 +33,7 @@ jobs:
     - name: Build FSharp.Formatting master
       run: cd FSharp.Formatting && .\build -t Build
     - name: Run fsdocs
-      run: FSharp.Formatting\src\FSharp.Formatting.CommandTool\bin\Release\netcoreapp3.1\fsdocs.exe build --sourcefolder fsharp  --parameters  fsdocs-navbar-position fixed-left
+      run: FSharp.Formatting\src\FSharp.Formatting.CommandTool\bin\Release\netcoreapp3.1\fsdocs.exe build --sourcefolder fsharp  --parameters  fsdocs-navbar-position fixed-left root /fsharp-core-docs/
```

Comment:
Basically, if F#F isn't doing the right thing for this repo without parameters, it's a bug in F#F :-)